### PR TITLE
[MRG] Coregistration

### PR DESCRIPTION
**Pull request by teonlamont**
closes https://github.com/mne-tools/mne-python/issues/2235

Allows for coregistration to take any instance that has info (Epochs, Evoked, Raw)

### CODE REVIEW

**Comment by teonlamont** [comment]
will add option to use a Inst object.

**Comment by teonlamont** [comment]
Actually, adding Inst object support will be a separate pr. This is now ready for review

**Comment by agramfort** [comment]
looks reasonable to me

File: mne/gui/__init__.py [hunk 1]
```diff
@@ -35,13 +35,17 @@ def coregistration(tabbed=False, split=True, scene_width=0o1, raw=None,
         unnecessary for wx backend).
     scene_width : int
         Specify a minimum width for the 3d scene (in pixels).
-    raw : None | str(path)
-        Path to a raw file containing the digitizer data.
+    inst : None | str(path)
```
Comment (Eric89GXL) [inline mne/gui/__init__.py]: we don't usually put `(path)` in there, go ahead and remove it

File: mne/gui/__init__.py [hunk 2]
```diff
@@ -35,13 +35,17 @@ def coregistration(tabbed=False, split=True, scene_width=0o1, raw=None,
...
+        Path to an instance file containing the digitizer data. Compatible for
+        Raw, Epochs, and Evoked files.
     subject : None | str
         Name of the mri subject.
     subjects_dir : None | path
         Override the SUBJECTS_DIR environment variable
         (sys.environ['SUBJECTS_DIR'])
+    raw : str(path)
+        This parameter has been deprecated for the `inst` parameter.
+        `raw` will be removed in 0.11.
```
Comment (Eric89GXL) [inline mne/gui/__init__.py]: You should remove this, and instead add this function name to `_deprecation_ignores` here:

https://github.com/mne-tools/mne-python/blob/master/mne/tests/test_docstring_parameters.py#L59

We don't want to have the deprecated parameters in the docstring anymore, they're just in the signature itself for completeness.

**Comment by teonlamont** [comment]
so this pr is good to go. I open an issue about a separate problem I discovered.

**Comment by agramfort** [comment]
can you see why travis is not happy?

@christianbrodbeck any objection?

besides LGTM

**Comment by teonlamont** [comment]
the builds were stopped. how do you restart a build?

**Comment by Eric89GXL** [comment]
You can do it from the Travis site but you might need to be owner of the repo, I'll restart

**Comment by agramfort** [comment]
thanks @teonlamont 

I assume that @christianbrodbeck was ok ;)

**Pull request merged**